OWML.ModHelper/ModManifest.cs

```diff
@@ -24,6 +24,9 @@ public class ModManifest : IModManifest
         [JsonProperty("owmlVersion")]
         public string OWMLVersion { get; private set; }
 
+        [JsonProperty("dependencies")]
+        public string[] Dependencies { get; private set; }
```

Comment:
if you remove "private" then you can set this from the outside and remove SetDependencies. Not a big deal, if you wanna keep this, then it's fine ;)

Reply:
yeah, just thought it'd be more in line with the setsettingsvalue in the config one

Reply:
I see why you need to set it - in case it's null.
You can set the default value (in case of null) like this:
 public string[] Dependencies { get; private set; } = new[] { "None" }
Then you can remove SetDependencies and the place you call SetDependencies :)